R/zzz.R

```diff
@@ -9,6 +9,12 @@
       ))
     }
   }
+  else {
+    strings <- strsplit(system("ldd --version | grep ldd", intern = TRUE), " +")[[1]]
+    if (as.numeric(tail(strings, n = 1)) < 2.35) {
```

Comment:
Because I am using v2.35 to compile, is it a requirement that the runtime Ubuntu also has to have v2.35 or newer?

Reply:
@Yuri05 do you know this?